mobile/src/bundles/common/components/radio-buttons/radio-buttons.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import {
+    type Control,
+    type FieldPath,
+    type FieldValues,
+} from 'react-hook-form';
+import RadioGroup, {
+    type RadioButtonProps,
+} from 'react-native-radio-buttons-group';
+
+import { Text, View } from '~/bundles/common/components/components';
+import { Color, TextCategory } from '~/bundles/common/enums/enums';
+import { useFormController } from '~/bundles/common/hooks/hooks';
+import { globalStyles } from '~/bundles/common/styles/styles';
+
+type Properties<T extends FieldValues> = RadioButtonProps & {
+    radioButtons: RadioButtonProps[];
+    label?: string;
+    name: FieldPath<T>;
+    control: Control<T, null>;
+};
```

Comment:
why do you have multiple RadioButtonProps ?

Reply:
We have additional props, for example, layout="row" from sign-up-form, so it goes through ...props. And radiobuttons, I need to add styles. Strange, but I can change the color if I set it in radioButtons.

Reply:
From your type, I see that I can pass RadioButtonProps via radioButtons prop and also directly, and the label is already in RadioButtonProps so it may cause conflicts

Reply:
> From your type, I see that I can pass RadioButtonProps via radioButtons prop and also directly, and the label is already in RadioButtonProps so it may cause conflicts

Let's fix this